### PR TITLE
Added "UseDecimalCurrency" to the config.yml & changed the rent sign

### PR DESCRIPTION
**Pull request by FabianGal45**
The changes I have made were some quick fixes.

I have first changed the way the rent sign looks when someone rents a claim as I did not like how the name of the player was not showing fully. This can create confusion when the owner of the claim wants to know who has rented the claim.
The changes are to the first 2 lines of the sign. I took away the title "[Real Estate]" and replaced it with "[Rented]" and removed the "rented by" from the second line to allow the name of the player to be shown as much as possible.

The second change (Disabling the decimal numbers) was made for me personally as I use items as currency and I did not like the look of the signs with decimal numbers as well as I did not want to create confusion when a player would sell a plot for a price with a decimal.

### CODE REVIEW

File: src/me/EtienneDx/RealEstate/Transactions/ClaimRent.java [hunk 1]
```diff
@@ -107,8 +122,8 @@ public boolean update()
 			else if(sign.getBlock().getState() instanceof Sign)
 			{
 				Sign s = (Sign) sign.getBlock().getState();
-				s.setLine(0, RealEstate.instance.config.cfgSignsHeader);
-				s.setLine(1, Utils.getSignString("Rented by " + Bukkit.getOfflinePlayer(buyer).getName()));
+				s.setLine(0, ChatColor.GOLD + "[Rented]"); //Changed the header to "[Rented]" so that it won't waste space on the next line and allow the name of the player to show underneath.
```
Comment (EtienneDx) [inline src/me/EtienneDx/RealEstate/Transactions/ClaimRent.java:125]: Could you move this line to a config line, and maybe make the same for the other options (sell & lease)

**Review by EtienneDx — COMMENTED** [review]
Looks great, I'm extremely busy currently but I'm glad someone is stepping up with the dev :)
I just have a question, what is the iml file and should it be commited?

File: src/me/EtienneDx/RealEstate/Transactions/ClaimRent.java (hunk 1 above)
Comment (FabianGal45) [reply src/me/EtienneDx/RealEstate/Transactions/ClaimRent.java:125]: My pleasure. 

To answer your question, no the iml file was not really supposed to be committed it kinda was made by my IDE (Intellij) automatically and I did not realize it at first. I should have probably removed it but I did not put that much thought into it tbh. I'll try to do get rid of it in the next commit and add the new things you suggested. 

Though that might take a week or so, as I have other priorities at the moment.

Comment (EtienneDx) [reply src/me/EtienneDx/RealEstate/Transactions/ClaimRent.java:125]: Alright thanks, I'll keep an eye on the PR :)

**Comment by FabianGal45** [comment]
So I started working on this with the idea that I will be able to add the changes to the sell and lease signs too. But after some checking, I saw that the sell and lease signs break once bought the claim so they don't need it.

In the next commit, I just removed the iml file. 

For me, it would not work to edit or build the jar without that iml file so if you don't use Intellij you might want to try and build the jar and see if it works for you just as a backup. Otherwise, it should be working fine.

**Comment by EtienneDx** [comment]
For the iml file, you can have it locaaly without problem, the only thing is that it should be added to the gitignore so that it doesn't get committed :)
I'll try building next week hopefully, I'm very overwhelmed this week :/

**Pull request merged**